lib/IRGen/GenClass.cpp

```diff
@@ -63,22 +63,7 @@ using namespace irgen;
 /// What reference counting mechanism does a class-like type have?
 ReferenceCounting irgen::getReferenceCountingForType(IRGenModule &IGM,
```

Comment:
I would just fold this function into callers

Reply:
Yeah. PR coming shortly addressing all of the feedback here. Thanks!